docs/source/queries/schema-agreement.md

```diff
@@ -87,4 +58,23 @@ if session.check_schema_agreement().await? {
 # }
 ```
 
+### Automated awaiting schema agreement
+
+The driver automatically awaits schema agreement after a schema-altering query is executed.
+Waiting for schema agreement more than necessary is never a bug, but might slow down applications which do a lot of schema changes (e.g. a migration).
+For instance, in case where somebody wishes to create a keyspace and then a lot of tables in it, it makes sense only to wait after creating a keyspace
+and after creating all the tables rather than after every query. Therefore, the said behaviour can be disabled:
 
+```rust
+# extern crate scylla;
+# use scylla::SessionBuilder;
+# use std::error::Error;
+# async fn check_only_compiles(session: &Session) -> Result<(), Box<dyn Error>> {
+let session = SessionBuilder::new()
+    .known_node("127.0.0.1:9042")
+    .auto_await_schema_agreement(false)
+    .build()
+    .await?;
+# Ok(())
+# }
+```
```

Comment:
I think it'd be better if this was at the start of this doc page (as a sort of warning: "You probably don't need to read the remaining portion of the doc - the driver does it for you")